docs/examples/puppeteer_recursive_crawl.ts

```diff
@@ -6,7 +6,7 @@ const crawler = new PuppeteerCrawler({
         console.log(`Title of ${request.url}: ${title}`);
 
         await enqueueLinks({
-            pseudoUrls: ['https://www.iana.org/[.*]'],
+            globs: ['https://www.iana.org/*'],
```

Comment:
i dont think this is the same, single `*` matches up until the first `/`, you need `**`, and maybe even without the initial slash (please test this, i am not 100% sure)

Reply:
Will test to make sure it still works, and correct the patterns otherwise

Reply:
When you are there, also check whether `https?://www.iana.org/` will match both https and http, I believe it should. Some examples where allowing both protocols, and we should show how to do that with globs too.

Reply:
So:

1. To have the same matching, it is with double `*`
2. `http?://www.iana.org/**` will match http, https and probably any variant of `http<letter>`. If that's fine I'll add it in the example that had optional https

Reply:
`http?` will not match `http` though (just checked)

Reply:
looks like `http?(s)://www.iana.org/**` is what we need here

Reply:
Check [here](https://www.digitalocean.com/community/tools/glob?comments=false&glob=http%3F%28s%29%3A%2F%2Fwww.iana.org%2F%2A%2A&matches=false&tests=http%3A%2F%2Fwww.iana.org&tests=https%3A%2F%2Fwww.iana.org&tests=httpa%3A%2F%2Fwww.iana.org&tests=http%3A%2F%2Fwww.iana.org%2F&tests=https%3A%2F%2Fwww.iana.org%2F&tests=httpa%3A%2F%2Fwww.iana.org%2F&tests=http%3A%2F%2Fwww.iana.org%2Fsub&tests=https%3A%2F%2Fwww.iana.org%2Fsub&tests=httpa%3A%2F%2Fwww.iana.org%2Fsub&tests=http%3A%2F%2Fwww.iana.org%2Fsub%2Fdomain&tests=https%3A%2F%2Fwww.iana.org%2Fsub%2Fdomain&tests=httpa%3A%2F%2Fwww.iana.org%2Fsub%2Fdomain)

Reply:
Will that match http *and* https? If so..that is some confusing syntax globs have 😅

Reply:
Sweet, thanks! Will change

Reply:
It is confusing indeed :)

Reply:
oh god, it means "one character" and not zero or one, wth

Reply:
maybe we should support `http?://` prefix automatically (rewrite it to the right form before we pass it to minimatch) :]